### PR TITLE
Resolve Vulnerabilities

### PR DESCRIPTION
**Pull request by EmilyZhang777**
This PR addresses the following vulnerabilities under test-site:
- bump webpack version to 5.90.0
- remove real live api key

J=VULN-37163,VULN-38428
TEST=auto

Ran `npm run test`. Spun up the test site and confirmed the pages are working.

### CODE REVIEW

**Review by benmcginnis — APPROVED** [review]

File: test-site/sample.env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_LIVE_API_KEY=[[Your Live API Key]]
```
Comment (nmanu1) [inline test-site/sample.env:1]: nit: can you also add a similar entry for `REACT_APP_MAPBOX_API_KEY`? looks like we forgot to add it previously for the test site

**Review by nmanu1 — COMMENTED** [review]
besides the nit, lgtm!

**Review by nmanu1 — COMMENTED** [review]
if we're making a new patch version for these fixes, we should probably merge into a hotfix branch rather than develop. but, personally I think we might be fine without because these are just changes to our test site

File: test-site/sample.env [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+REACT_APP_MAPBOX_API_KEY=[[Your Live API Key]]
```
Comment (nmanu1) [inline test-site/sample.env:2]: nit: this should probably read `Your Mapbox API Key` instead of live api key

**Pull request merged**